DFC.Swagger.Standard/SwaggerDocumentGenerator.cs

```diff
@@ -414,7 +420,7 @@ private void SetParameterType(Type parameterType, dynamic opParam, dynamic defin
                 
             }
 
-            if (inputType.Namespace == "System" && !isEnum || (inputType.IsGenericType && inputType.GetGenericArguments()[0].Namespace == "System"))
+            if (inputType.Namespace == "System" && (!isNullableEnum || !isEnum) || (inputType.IsGenericType && inputType.GetGenericArguments()[0].Namespace == "System"))
             {
                 if (paramType.Contains("System.String"))
```

Comment:
Why are you testing for this condition if you're not doing anything if it's true?